ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-ref.yaml

```diff
@@ -0,0 +1,48 @@
+ref:
+  as: hypershift-kubevirt-create
+  from: hypershift-operator
+  grace_period: 5m0s
+  timeout: 1h
+  cli: latest
+  env:
+    - name: HYPERSHIFT_NODE_COUNT
+      default: "3"
+      documentation: "The number nodes to automatically create and join to the cluster."
+    - name: HYPERSHIFT_NODE_MEMORY
+      default: "16"
+      documentation: "The amount of RAM that will be allocated for the kubevirt hosted cluster's nodes in Gi"
+    - name: HYPERSHIFT_NODE_CPU_CORES
+      default: "4"
+      documentation: "The number of CPU cores that will be assigned for the kubevirt hosted cluster's nodes."
+    - name: RUN_HOSTEDCLUSTER_CREATION
+      default: "true"
+      documentation: |-
+        Indicates whether this step should be skipped. Useful when it is a part of a workflow used in various
+        branches, and some branches don't need to execute that step.
+    - name: KUBEVIRT_CSI_INFRA
+      default: ""
+      documentation: |-
+        The default storage class on the infra cluster to be used by the guest cluster with the
+        kubevirt csi-driver.
+    - name: ENABLE_ICSP
+      default: "false"
+      documentation: "if true, add image content sources config(path=${SHARED_DIR}/mgmt_iscp.yaml)"
+    - name: ETCD_STORAGE_CLASS
+      default: ""
+      documentation: |-
+        The storage class that will be used for etcd on the hosted control plane.
+    - name: MCE_VERSION
+      default: ""
+      documentation: "version of the mce, if installed. (\"\", 2.2, 2.3, 2.4, 2.5)"
```

Comment:
`default: ""` here is doing the trick: if no value is supplied, `""` will be used and the `if [[ -n ${MCE_VERSION} ]] ;` will skip the MCE specific branches.
We can also detect it at runtime in the script, but it could be more error prone than simply checking a string.

Reply:
I think we should add some documentation if "" doesn't pass through MCE installation.

Reply:
done